neptune/__init__.py

```diff
@@ -52,7 +53,19 @@
 ANONYMOUS_API_TOKEN = constants.ANONYMOUS_API_TOKEN
 
 
-def init(project_qualified_name=None, api_token=None, proxies=None, backend=None):
+CURRENT_KWARGS = ('project', 'run', 'custom_run_id', 'mode')
```

Comment:
Shouldn't we have more of new-style `init` arguments?
* `custom_run_id`
* `mode`
* `name`
* `description`
* `tags`
* and so on...

Reply:
I was aiming for a "heuristic" approach: detect args that you can realistically expect to be used each time.

But maybe I'm too cunning for my own good. I'll add more args.